internal/pkg/cli/pipeline_init.go

```diff
@@ -552,7 +553,19 @@ func (o *initPipelineOpts) secretName() string {
 }
 
 func (o *initPipelineOpts) pipelineName() string {
```

Comment:
Let's just keep things dumb and less prone to bugs, realistically folks won't have huge app names, if anything it will be the repo name so i think we can just do:
```
if len name <= 100:
   return name
return name[:100]
```

Reply:
done